ambry-cloud/src/main/java/com.github.ambry.cloud/CloudBlobMetadata.java

```diff
@@ -28,6 +28,9 @@
   public static final String FIELD_EXPIRATION_TIME = "expirationTime";
   public static final String FIELD_ACCOUNT_ID = "accountId";
   public static final String FIELD_CONTAINER_ID = "containerId";
+  public static final String FIELD_ENCRYPTION_ORIGIN = "encryptionOrigin";
```

Comment:
Is `initiator` better than `origin` in this context?

Reply:
Either seems okay.  I prefer origin.